### ISSUE TITLE
Pass FIREBASE_TOKEN in firestore:delete

### ISSUE THREAD

**Issue by travwritescode** [prescottprue/cypress-firebase#40]
When using the `callFirestore('delete')` command, is there currently a way to pass the firebase token generated from `firebase-tools login:ci` as an option? Currently the command looks as though passing an options object does nothing as the passed in object is ignored while a new one is created with Cypress and args arguments: `const deleteArgsWithDefaults = addDefaultArgs(Cypress, args);`

In my specific use case I am attempting `callFirestore('delete', 'actionPath/in/Firestore', { '--token', Cypress.env.FIREBASE_TOKEN }` but the options object is being dropped and the console output when the call fails is as follows:

`CypressError: cy.exec('$(npm bin)/firebase firestore:delete actionPath/in/Firestore -P  -y --shallow') failed because the command exited with a non-zero code.`

So the options object `{ --token, Cypress.env.FIREBASE_TOKEN }` is being ignored for the favored default arguments. If it helps at all, I think the line in question is in `buildFirestoreCommand.js` with the line `const deleteArgsWithDefaults = addDefaultArgs(Cypress, args);` which creates a brand new object instead of building on the one that was passed in.

**Comment by prescottprue** [comment]
{assing in the options object with `args` should allow this to be passed along, like so:

```js
const options = { args: [ '--token', Cypress.env('FIREBASE_TOKEN')] }
callFirestore('delete', 'actionPath/in/Firestore', options)
```

That said, I am looking into automatically including the token if it is set to the Cypress environment

**Comment by prescottprue** [comment]
Released in v0.2.2
